README.md

```diff
@@ -55,6 +55,15 @@ You can also use this script developed by [enoch85](https://github.com/enoch85):
 ```
 #!/bin/bash
 
+# Install git if not existing
+if [ "$(dpkg-query -W -f='${Status}' "git" 2>/dev/null | grep -c "ok installed")" == "1" ]
+then
+    sleep 0.1
```

Comment:
Is there any reason for the sleep? If you use `if [ "$(dpkg-query -W -f='${Status}' "git" 2>/dev/null | grep -c "ok installed")" != "1" ]` then you can drop the sleep.